bitnami/victoriametrics/templates/NOTES.txt

```diff
@@ -0,0 +1,168 @@
+CHART NAME: {{ .Chart.Name }}
+CHART VERSION: {{ .Chart.Version }}
+APP VERSION: {{ .Chart.AppVersion }}
+
+Did you know there are enterprise versions of the Bitnami catalog? For enhanced secure software supply chain features, unlimited pulls from Docker, LTS support, or application customization, see Bitnami Premium or Tanzu Application Catalog. See https://www.arrow.com/globalecs/na/vendors/bitnami for more information.
+
+** Please be patient while the chart is being deployed **
+
+The following controllers have been deployed:
+
+  - vmselect
+  - vminsert
+  - vmstorage
+  {{- if .Values.vmauth.enabled }}
+  - vmauth
+  {{- end }}
+  {{- if .Values.vmagent.enabled }}
+  - vmagent
+  {{- end }}
+  {{- if .Values.vmalert.enabled }}
+  - vmalert
+  {{- end }}
```

Comment:
Missing the block below:
```
{{- if .Values.diagnosticMode.enabled }}
The chart has been deployed in diagnostic mode. All probes have been disabled and the command has been overwritten with:
(...)
```

Reply:
It's a scratch container so we cannot have diagnosticMode